### PR TITLE
add patches for minimus2 in AMOS-3.1.0

### PR DESCRIPTION
**Pull request by laraPPr**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by laraPPr** [comment]
Test report by @laraPPr
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node4005.donphan.os - Linux RHEL 8.8 (Ootpa), x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 545.23.08, Python 3.11.3
See https://gist.github.com/laraPPr/1ab127facbb89756fc84f23885723989 for a full test report.

**Comment by laraPPr** [comment]
@boegelbot please test @ generoso

**Comment by laraPPr** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/a/AMOS/AMOS-3.1.0_link_correctly_to_DELTAFILER_SHOWCOORDS.patch [hunk 1]
```diff
@@ -0,0 +1,14 @@
+minimus2 needs delta-filter and show-coords but these were not set correctly like NUCMER
```
Comment (boegel) [inline easybuild/easyconfigs/a/AMOS/AMOS-3.1.0_link_correctly_to_DELTAFILER_SHOWCOORDS.patch:1]: @laraPPr Can you please add yourself as author in both patches, and then fix the checksums accordingly?

```suggestion
minimus2 needs delta-filter and show-coords but these were not set correctly like NUCMER
author: Lara Peeters (HPC-UGent)

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3136.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/8d5139bae6960052849fefe39e84c0b3 for a full test report.

**Pull request merged**